tests/conftest.py

```diff
@@ -78,6 +79,14 @@
 asyncio.set_event_loop_policy = lambda policy: None
 
 
+def _utcnow():
+    """Make utcnow patchable by freezegun."""
+    return datetime.datetime.now(datetime.timezone.utc)
+
+
+dt_util.utcnow = _utcnow
```

Comment:
Was this needed due to partial() change or was it always broken?

Reply:
Its only because of the `partial`.  freezegun probably could use some logic similar to https://github.com/home-assistant/core/blob/45eb1efc6f680a8b4b926557fb09358d009a40ae/homeassistant/core.py#L233 but its probably too much of a corner case to upstream